front/src/View/Pages/Inscription.tsx

```diff
@@ -0,0 +1,197 @@
+                                <InscriptionInput id={"inputNom"} inputType={"text"} placeholder={"Nom"} className={""} onChange={this.handleNomChange}/>
+                                <InscriptionInput id={"Pseudo"} inputType={"text"} placeholder={"Pseudo"} className={""} onChange={this.handlePseudoChange}/>
+                            </div>
+
+                            <div className={"buttons-bottom"}>
+                                <InscriptionInput id={"inputMail"} inputType={"email"} placeholder={"Adresse email"}
+                                                  className={""} onChange={this.handleEmailChange}/>
+                                <InscriptionInput id={"inputPassword"} inputType={"password"}
+                                                  placeholder={"Mot de passe"}
+                                                  className={"custom"} onChange={this.handlePasswordChange}/>
+                                <InscriptionInput id={"inputPasswordConfirmation"} inputType={"password"}
+                                                  placeholder={"Confirmation mot de passe"}
+                                                  className={"custom"} onChange={this.handlePasswordChange}/>
+                                <div className={"submit-button"}>
+                                    <ButtonInscription/>
+                                </div>
+                            </div>
+                        </form>
+                        <br/>
+                        Déja Inscrit ? <a id={"redirect-button"} onClick={this.redirectToConnectionPage}>Connecte toi ! </a>
+                    </div>
+                </div>
+            </section>
+        )
+    }
+}
+
+interface InscriptionInputProps {
+    id: string;
+    inputType: string;
+    placeholder: string;
+    className: string;
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+}
+
+class InscriptionInput extends React.Component<InscriptionInputProps, {}> {
+
+    private readonly className: string;
+
+    constructor(props: InscriptionInputProps) {
+        super(props);
+        this.className = "form-control text-center mt-0 rounded-1 " + this.props.className;
+    }
+
+    public render(): ReactNode {
+        return <div className="form-group">
+            <input type={this.props.inputType}
+                   className={this.className}
+                   id={this.props.id}
+                   placeholder={this.props.placeholder}
+                   onFocus={e => e.target.placeholder = ""}
+                   onBlur={e => e.target.placeholder = this.props.placeholder}
+                   onChange={this.props.onChange}
+            />
+        </div>
+
+    }
+}
+
+class ButtonInscription extends React.Component<{}, {}> {
+    public render() {
+        return (
+            <button type="submit" className="btn btn-default btn-transparent mt-0 rounded-1 custom">Connexion</button>
```

Comment:
Should'nt it be "Créer mon compte" or something like this ?

Reply:
done !